nettle/station_set.py

```diff
@@ -488,16 +488,18 @@ def _get_old_metadata(self):
 
         return old_station_metadata, old_stations, old_hash
 
-    def station_metadata_to_geojson(self, data, **kwargs):
+    def station_metadata_to_geojson(self, data: dict, md_path:str, **kwargs):
```

Comment:
I think removing MetadataHandler.STATION_METADATA_FILE_NAME in line 475 will break the currently etls who don't have md_path set. Setting md_path default to MetadataHandler.STATION_METADATA_FILE_NAME will solve this

Reply:
unfortunately you can’t set a default to = an instance variable. Hence the hacky backwards compatibility for `data`. I’ll set something similar up for `md_path`.